specification/0081-active-user-beacon.md

```diff
@@ -0,0 +1,43 @@
+## Summary
+
+This RFC describes a privacy-friendly mechanism through which the loader can signal themselves as an active user, in order to provide MAU (Monthly Active User) metrics. 
+
+## Motivation
+
+When discussing with partners, requesting grants or negotiating sponsorships, it is quite important for The Quilt Project to be able to give accurate MAU metrics. 
+The choice to not include any metadata in the signal was made to keep the mechanism privacy-friendly, as Quilt collects zero personal data or usage statistics.
+
+## Explanations
+
+On launch, the loader will start the following process asynchronously:
+1. Check if the `disableBeacon` property or the `QUILT_DISABLE_BEACON` environment variable is set. If so, the process is aborted (the launch process is unaffected).
+2. Check for the last signalled month in a standard persistent location, if it is equal to the current month the process is aborted.
+3. Send a POST request to `https://beacon.quiltmc.org/signal` without any specific body or headers.
+4. Save the current month into the same standard persistent location as step 2.
+
+This process and opt-out methods MUST be clearly documented through the appropriate means.
+
+## Beacon Server Restrictions
+
+The server hosted at `beacon.quiltmc.org` MUST:
+1. Be open-source.
+2. Be auditable by any Quilt developer who wish to, within reason.
+3. Not store any data present inside the request.
+4. Not process any data present inside the request beside what is strictly required to establish the connection, following current best practices and technical feasibility. 
+
+## Drawbacks
+
+This process can be seen as privacy-breaking by some. We implemented an opt-out and decided to not collect any data to minimize some of that concern. 
```

Comment:
Shouldn't this be opt-_in_?

Reply:
If it was opt-in then you wouldn't count the active users. You'd count the people that care enough about Quilt to report their own presence, and that's a much smaller group, you'd vastly underreport the number of active users.